espressif/vscode-esp-idf-extension
# Fix headers and include resolution by specifying compileCommand.json path

## Description

Added compileCommands.json path, this will help getting rid of the squiggly lines in C files for the esp projects after build.

Task: https://jira.espressif.com:8443/browse/VSC-1234

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Create new esp project
2. Build project
3. Open the main c file of the project (Ex for blink project: blink_example_main.c)

- Expected behaviour:
The file should display no errors (no squiggly red lines)

## How has this been tested?

Created esp projects using all possible options in VS Code Extension on Windows 11 with no issues shown after building the projects.

**Test Configuration**:
* ESP-IDF Version: 5.1.2
* OS (Windows,Linux and macOS): Windows 11

## Checklist
